docs/cc-fuzzing-rules.md

```diff
@@ -93,6 +94,8 @@ documentation for the set of targets generated.
 | <a id="fuzzing_decoration-engine"></a>engine |  The label of the fuzzing engine used to build the binary.   |  none |
 | <a id="fuzzing_decoration-corpus"></a>corpus |  A list of corpus files.   |  <code>None</code> |
 | <a id="fuzzing_decoration-dicts"></a>dicts |  A list of fuzzing dictionary files.   |  <code>None</code> |
-| <a id="fuzzing_decoration-tags"></a>tags |  Tags set on the fuzzing regression test.   |  <code>None</code> |
+| <a id="fuzzing_decoration-instrument_binary"></a>instrument_binary |  If true, depend on the <code>raw_binary</code> through a Bazel   configuration transition that uses configuration flags from the   <code>@rules_fuzzing//fuzzing</code> package to determine fuzzing build mode,   engine, and sanitizer instrumentation. If false, assumes that   <code>raw_binary</code> is already built in the proper configuration.   |  <code>True</code> |
```

Comment:
Should this mention that users probably don't need to use this (and/or link to somewhere that they can understand why this is there?)

Reply:
You're right - I think we can go even further and just ask users to reach out to us if they think the default instrumentation mode doesn't work for them. I clarified this in the docs.